test/unit/spec/IDPDiscovery_spec.js

```diff
@@ -134,6 +134,59 @@ function (Q, OktaAuth, LoginUtil, Okta, Util, AuthContainer, IDPDiscoveryForm, B
     return name;
   }
 
+  function setupWithCustomButtons() {
+    var settings = {
+      customButtons: [
+        {
+          title: 'test text',
+          className: 'test-class',
+          click: function (e) {
+            $(e.target).addClass('new-class');
+          },
+          dataAttr: 'test-data'
+        }
+      ]
+    };
+    return setup(settings);
+  }
+
+  function setupWithCustomButtonsWithIdp() {
+    var settings = {
+      customButtons: [
+        {
+          title: 'test text',
+          className: 'test-class',
+          click: function (e) {
+            $(e.target).addClass('new-class');
+          }
+        }
+      ],
+      idps: [
+        {
+          type: 'FACEBOOK',
+          id: '0oaidiw9udOSceD1234'
+        }
+      ]
+    };
+    return setup(settings);
+  }
+
+  function setupWithCustomButtonsWithoutIdp() {
```

Comment:
I think that this is equivalent to `setupWithCustomButtons`, so we can probably remove it and use that one instead